templates/signup_page.html

```diff
@@ -53,6 +53,18 @@ <h1> Vacation Planner </h1>
         </form>
     </div>
 
+    <div class="d-flex justify-content-center align-items-center">
```

Comment:
nit: we can consider using warnings over alerts in Bootstrap.
https://getbootstrap.com/docs/5.0/components/alerts/

Reply:
Done, the only thing is user cannot remove the banner